recipe/meta.yaml

```diff
@@ -59,6 +63,7 @@ requirements:
     - r-scales
     - r-tidyr >=0.6.1
     - r-xts
+    - r-bh
```

Comment:
r-bh is headers only and so only required during compilation in the host section and can be removed from run section.